v20.2/restore.md

```diff
@@ -60,7 +60,7 @@ You can control `RESTORE` behavior using any of the following in the `restore_op
 `skip_missing_sequence_owners`                                      | N/A                                         | <span class="version-tag">New in v20.2:</span> Must be used when restoring either a table that was previously a [sequence owner](create-sequence.html#owned-by) or a sequence that was previously owned by a table.<br><br>Example: `WITH skip_missing_sequence_owners`
 `skip_missing_views`                                                | N/A                                         | Use to skip restoring [views](views.html) that cannot be restored because their dependencies are not being restored at the same time.<br><br>Example: `WITH skip_missing_views`
 `encryption_passphrase`                                             | Passphrase used to create the [encrypted backup](take-and-restore-encrypted-backups.html) |  The passphrase used to decrypt the file(s) that were encrypted by the [`BACKUP`](take-and-restore-encrypted-backups.html) statement.
-`DETACHED`                                                          | N/A                                         | <span class="version-tag">New in v20.2:</span> When `RESTORE` runs with `DETACHED`, the job will execute asynchronously and the job ID will be returned immediately without waiting for the job to finish. Note that with `DETACHED` specified, further job information and the job completion status will not be returned. For more on the differences between the returned job data, see the [example](restore.html#restore-a-backup-asynchronously) below. To check on the job status, use the [`SHOW JOBS`](show-jobs.html) statement. <br><br>To run a restore within a [transaction](transactions.html), use the `DETACHED` option.
+`DETACHED`                                                          | N/A                                         | <span class="version-tag">New in v20.2:</span> When `RESTORE` runs with `DETACHED`, the job will execute asynchronously. The job ID is returned after the backup completes. Note that with `DETACHED` specified, further job information and the job completion status will not be returned. For more on the differences between the returned job data, see the [example](restore.html#restore-a-backup-asynchronously) below. To check on the job status, use the [`SHOW JOBS`](show-jobs.html) statement. <br><br>To run a restore within a [transaction](transactions.html), use the `DETACHED` option.
```

Comment:
Here need to change "backup" here to just "job" or "restore"... so:

"The job ID is returned after the restore's job creation completes."

This goes for all of these sentences in the restore docs.

Reply:
Oops thanks for catching that Kathryn! Fixed now